src/api.js

```diff
@@ -844,6 +962,63 @@ Module["onRuntimeInitialized"] = function onRuntimeInitialized() {
         return stmt;
     };
 
+    /** Iterate over multiple SQL statements in a SQL string
+     @param {string} sql a string of SQL
+     @return {StatementIterator} the resulting statement iterator
+     */
+    Database.prototype["iterateStatements"] = function iterateStatements(sql) {
+        return new StatementIterator(sql, this);
+    };
+
+    /* Internal methods to implement statement iteration */
+
+    Database.prototype.advanceIterator = function advanceIterator(iter) {
+        if (iter.sqlPtr === null) {
+            throw "Invalid iterator";
+        }
+        if (iter.activeStatement !== null) {
+            iter.activeStatement["free"]();
+            iter.activeStatement = null;
+        }
+        if (!this.db) {
+            this.finalizeIterator(iter);
+            throw "Database closed";
```

Comment:
```suggestion
            throw new Error("Database closed");
```

Reply:
I applied this change... it is now inconsistent with the code in Database.run() and Database.exec() - should I modify those throws for consistency?  (Probably other throws as well...)

Reply:
Unfortunately this would be a breaking change. This is one we will have to do at some point, but let's keep that for later.